src/styles/index.scss

```diff
@@ -1,3 +1,131 @@
+$body-font: "Roboto", sans-serif;
+$font-size: 10px;
+$color-main-accent: #060b35;
+$color-secondary: #616070;
+$color-blue-accent: #00acdc;
+
 body {
   margin: 0;
+  font-family: $body-font;
+  font-size: $font-size;
+}
+
+.card {
```

Comment:
according to [checklist](https://github.com/mate-academy/layout_product-cards/blob/master/checklist.md) point 11 there should be 1 file for each BEM block - so 1 for stars and 1 for card